### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by vikleer**
Update the game over message so people know how to play again!

### CODE REVIEW

**Comment by vikleer** [comment]
I have checked the changes and everything is fine

**Review by eatteer — APPROVED** [review]
What the duck

**Review by vikleer — COMMENTED** [review]
It's fine

File: index.html [hunk 1]
```diff
@@ -47,7 +47,7 @@ <h1>Tic Tac Toe</h1>
   </tr>
 </table>
 
-<h2 hidden>Game over</h2>
+<h2 hidden>Game over, refresh to play again 🧑‍💻 🤖!</h2>
```
Comment (vikleer) [inline index.html:51]: You can say the user "Good luck"

Comment (vikleer) [inline index.html:51]: You can say the user "Good luck"

**Review by vikleer — COMMENTED** [review]
jasd

**Pull request merged**